shm/cargo.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <string>
+
+class Cargo {
+  private: 
+     std::string name_;
+     size_t amount_;
+     size_t base_price_;
```

Comment:
We decided on Discord to use camelCase for functions and variables

```suggestion
     size_t basePrice_;
```

Reply:
Please change it everywhere